src/utils.ts

```diff
@@ -35,5 +35,34 @@ export function getRequiredRosDistributions(): string[] {
 			RegExp("\\s")
 		);
 	}
+
+	if (!validateDistro(requiredRosDistributionsList)) {
+		throw new Error("Input has invalid distribution names.");
+	}
+
 	return requiredRosDistributionsList;
 }
+
+//list of valid linux distributions
+const validDistro: string[] = [
+	"kinetic",
+	"lunar",
+	"melodic",
+	"noetic",
+	"dashing",
+	"eloquent",
+	"foxy",
+];
+
+//go through requiredRosDistributionsList and validate every linux distribution input
```

Comment:
This comment describes code and not functionality. Alternative suggestion: "Determine whether all inputs name supported ROS distributions."